docs/markdown/authoritative/modes-of-operation.md

```diff
@@ -85,7 +85,26 @@ PowerDNS supports multiple masters. For the BIND backend, the native BIND
 configuration language suffices to specify multiple masters, for SQL based backends,
 list all master servers separated by commas in the 'master' field of the domains table.
 
-Since version 4.0.0, PowerDNS requires that masters sign their notifications. During transition and interoperation with other nameservers, you can use options **allow-unsigned-notify** to permit unsigned notifications. For 4.0.0 this is turned off by default, but it might be turned on permanently in future releases.
+Since version 4.0.0, PowerDNS requires that masters sign their
+notifications.  During transition and interoperation with other nameservers,
+you can use options **allow-unsigned-notify** to permit unsigned
+notifications.  For 4.0.0 this is turned off by default, but it might be
```

Comment:
This comment is the wrong way around (still). the setting is set to 'yes' now, but might be 'no' later